### ISSUE TITLE
make Vue Authenticator UI Responsive

### ISSUE THREAD

**Issue by kannieWeesNie** [aws-amplify/docs#396]
it would be nice to have the vue authenticator components ui responsive.

i tried using the authenticator, it displays great sign-in form but it is not responsive. i tested in chrome dev tools with various mobile displays both landscape and portrait and the authenticator form does not fit on small screens.

i also tried the authenicator form inside a vuetify card with the same result, it is not responsive and does not fit inside small device screens in either orientation (some of the form is off-screen).

so, to sum up, it would be great if all the amplify authentication components could be responsive and compatible with vuetify.

thanks

**Comment by mfreeman451** [comment]
@kannieWeesNie it is pretty easy to create your own form and styling, take a look at https://github.com/mfreeman451/quasar-amplify-demo

**Comment by kannieWeesNie** [comment]
@mfreeman451 thanks. the amplify login, etc screens are behaving responsively out the box. something weird must have been going on with my development machine to cause them not to but its all fine now.

**Comment by mfreeman451** [comment]
maybe you can close this then?
